sql_generators/fenix_use_counters/__init__.py

```diff
@@ -0,0 +1,109 @@
+import os
+import shutil
+from pathlib import Path
+
+import click
+import yaml
+from jinja2 import Environment, FileSystemLoader
+
+from bigquery_etl.cli.utils import use_cloud_function_option
+from bigquery_etl.format_sql.formatter import reformat
+from bigquery_etl.schema import SCHEMA_FILE, Schema
+from bigquery_etl.util.common import write_sql
+
+FILE_PATH = Path(os.path.dirname(__file__))
+BASE_DIR = Path(FILE_PATH).parent.parent
+TEMPLATE_CONFIG = FILE_PATH / "templates/templating.yaml"
+
+
+def generate_query(project, dataset, destination_table, write_dir):
+    """Generate feature usage table query."""
+    with open(TEMPLATE_CONFIG, "r") as f:
+        render_kwargs = yaml.safe_load(f) or {}
+    env = Environment(loader=FileSystemLoader(str(FILE_PATH / "templates")))
+    template = env.get_template("query.sql")
+
+    write_sql(
+        write_dir / project,
+        f"{project}.{dataset}.{destination_table}",
+        "query.sql",
+        reformat(template.render(**render_kwargs)),
+    )
+
+
+def generate_view(project, dataset, destination_table, write_dir):
```

Comment:
This could also be a template `view.sql` file, so it could be reused for firefox desktop.

Reply:
Not sure I'm following - what do you mean?